vars/multicloudDeploy.groovy

```diff
@@ -44,7 +47,8 @@ void call(parameters = [:]) {
 
         ConfigurationHelper configHelper = ConfigurationHelper.newInstance(this)
             .loadStepDefaults()
-            .mixinGeneralConfig(script.commonPipelineEnvironment, GENERAL_CONFIG_KEYS)
+            .mixinGeneralConfig(script.commonPipelineEnvironment, GENERAL_CONFIG_KEYS, CONFIG_KEY_COMPATIBILITY)
+            .mixinStepConfig(script.commonPipelineEnvironment, STEP_CONFIG_KEYS, CONFIG_KEY_COMPATIBILITY)
             .mixin(parameters, PARAMETER_KEYS)
```

Comment:
Could you also please merge in the stage configuration. We will need that later on. Maybe even now. 
It would be create if you could remove passing stage configuration as parameter to this step but letting the step resolve the stage config instead. 

https://github.com/SAP/cloud-s4-sdk-pipeline-lib/blob/master/vars/stageProductionDeployment.groovy#L60
Would be great to get rid of passing the parameters cfTargets, neoTargets, cfCreateServices. 
But can also be done in a follow up.